### PR TITLE
Fix: Hover active color is not correct in Tabs widget [ED-10305]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 1]
```diff
@@ -433,26 +433,32 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 		await editor.addWidget( 'nested-tabs', container );
 		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
 
+		await page.pause();
```
Comment (hein-obox) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:436]: Remove.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 2]
```diff
@@ -253,7 +253,7 @@
 :is( .elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-n-tab-title,
 	.elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content .e-n-tab-title ) {
 
-	&.e-normal:hover {
+	&:hover {
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:256]: Have you tried this? 

With doing this, we are targetting the `e-normal` directly, without changing the specifity. 

And we are remove the hover effect from the tabs `.e-n-tabs-content`.

```.elementor .elementor-element.elementor-widget-n-tabs > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-normal:hover {
		background-color: var( --n-tabs-title-background-color-hover );
		background-image: initial;
	}
```

Comment (davseve) [reply modules/nested-tabs/assets/scss/frontend.scss:256]: I tried that snippet, it makes no change.
We can take it offline and investigate it together.
Removing the `.e-normal` class decreases the specificity and allows the `:not(:hover)` rule to be assigned to the active tab.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 1 above)
Comment (davseve) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:436]: Removed.
My mistake

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 3]
```diff
@@ -416,7 +416,7 @@ protected function register_controls() {
 				'name' => 'tabs_title_background_color',
 				'types' => [ 'classic', 'gradient' ],
 				'exclude' => [ 'image' ],
-				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title:not( .e-active ):not( :hover )",
+				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title:not( .e-active )",
```
Comment (hein-obox) [inline modules/nested-tabs/widgets/nested-tabs.php:419]: Does this work correctly for both mobile and desktop? I assume we do need the `:not( :hover )` for the desktop version.

**Comment by hein-obox** [comment]
@davseve Can you please add a mindmap, to make sure that we cover all scenarios.

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 3 above)
Comment (davseve) [reply modules/nested-tabs/widgets/nested-tabs.php:419]: Correct, 
The hover worked but something else broke.
I fixed it now.

**Review by hein-obox — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**